src/ui/post_card/middle_post_card/middle_post_card.tsx

```diff
@@ -4,6 +4,7 @@ import { useState } from 'react';
 import { styled } from 'styled-components';
 import { useAppDispatch } from '../../../hook';
 import { PostCardModel } from '../post-card.model';
+import { FavoritePostIcon } from '#features/favorite-icon/favorite-icon';

```

Comment:
UI component can't depend on feature. Possible alternatives:

1) pass `FavoritePostIcon` component as prop outside

2) onClick logic & process adding to bookmark outside